### DISCUSSION TITLE
为什么连接服务器出错

### DISCUSSION THREAD

**Discussion by TiankY-W** [FloatTech/ZeroBot-Plugin#381] [Q&A] [answered]
![WJH {CQ}ORLAAG_0C8DAIRJ](https://user-images.githubusercontent.com/111638198/185745411-1c5cfb32-faae-4132-8492-2ae8c3bc70e6.png)

**Comment by Jiang-Red** [comment, marked as answer]
gocq和zbp是否同时运行
gocq是否选择的是正向ws
